README.md

```diff
@@ -13,37 +13,147 @@ Our focus is to explore a variety of software applications to ensure we have goo
+
+## Getting Started
+
+This Cinema 4D integration for AWS Deadline Cloud has two components that you will need to install:
+
+1. The Cinema 4D submitter extension must be installed on the workstation that you will use to submit jobs; and
+2. The Cinema 4D adaptor must be installed on all of your AWS Deadline Cloud worker hosts that will be running the Cinema 4D jobs that you submit.
+
+Before submitting any large, complex, or otherwise compute-heavy Cinema 4D render jobs to your farm using the submitter and adaptor that you
+set up, we strongly recommend that you construct a simple test scene that can be rendered quickly and submit renders of that scene to your farm to ensure that your setup is correctly functioning.
+
 
 ## Submitter
 
-This package provides a Cinema 4D plugin that creates jobs for AWS Deadline Cloud using the [AWS Deadline Cloud client library][deadline-cloud-client]. Based on the loaded scene it determines the files required, allows the user to specify render options, and builds an [OpenJD template][openjd] that defines the workflow.
+The Cinema 4D submitter extension creates a button in your Cinema 4D UI (under Extensions tab) that can be used to submit jobs to AWS Deadline Cloud. Clicking this button reveals a UI to create a job submission for AWS Deadline Cloud using the [AWS Deadline Cloud client library][deadline-cloud-client].
+It automatically determines the files required based on the loaded scene, allows the user to specify render options, builds an
+[Open Job Description template][openjd] that defines the workflow, and submits the job to the farm and queue of your chosing.
+
+### To install the submitter extension:
+
+#### Prerequisites
+
+1. Install the required python modules:
+
+```
+pip install deadline-cloud-for-cinema-4d
+pip install deadline[gui]
+```
+
+2. Set up the `C4DPYTHONPATH311` environment variable:
+  - Windows: 
+```
+set C4DPYTHONPATH311"Path/to/site-packages"
+```
+
+  - Mac:
+```
+export C4DPYTHONPATH311="Path/to/site-packages"
+```
+
+#### Downloading the extension
+
+The Cinema 4D extension can be downloaded from the git repo:
+
+https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/blob/mainline/deadline_cloud_extension/DeadlineCloud.pyp
+
+
+#### Set up Cinema 4D to access DeadlingCloud extension: 
+
+There are 2 ways to setup Cinema 4D to access this extension:
+
+1. Plugin directory method:
+    1. Create a new folder "plugins" within the Cinema 4D installation directory. 
+    2. Place the `DeadlineCloud.pyp` within this "plugins" directory. 
+
+2. Environment variable method:
+    1. Add the `g_additionalModulePath` environment variable to point to the location where `DeadlineCloud.pyp` exists, so that Cinema 4D can load the plugin.
+
+Windows:
+```
+set g_additionalModulePath="Path\to\DeadlineCloud.pyp"
+```
+
+Linux or Mac:
+```
+export g_additionalModulePath="Path\to\DeadlineCloud.pyp"
+```
```

Comment:
The path should use forward slashes.

Reply:
Oh makes sense, I'll update this right away.